docs/commands/deck.md

```diff
@@ -0,0 +1,38 @@
+# `/deck` command
+
+Display the contents of a deck in Discord! _This command is in development._
+
+## Definitions
+
+**YDKE**: A storage format for Yu-Gi-Oh! decks that can be exported from various deck building applications. They look like URLs, beginning with `ydke://`. An example of a YDKE URL looks like this: `ydke://1xqfAdcanwHXGp8Bd+lHBHfpRwR36UcEX8ZAAl/GQAJfxkACYx3EAzA20AMwNtADMDbQA/liqgQk6AgEJOgIBCToCASxxDUDscQ1A7HENQPw6KgA8OioAPDoqADAqZQA/YmcBf2JnAX9iZwFppv0Aaab9AGmm/QBgAhAAsWp2gTvJf4EkWK+BZFivgWRYr4FBlTqBQZU6gUGVOoFPqRxAcLHcgHCx3IBwsdyAQ==!Vi0OBRXfCwVs1ZQA8GQuALiBCQPBGnIE/gDeA2jWawV9vZ8DoBOYBIPjPgI1tL4EhGVOAbXg4AQBroQB!sjLMBbIyzAWyMswFQ77dAEO+3QBDvt0AJpBCAyaQQgMmkEIDhCV+AIQlfgCEJX4Ab3bvAG927wCBFq4D!`.
+
+## Parameters
+
+Name | Required? | Description | Type
+--- | --- | --- | ---
+`deck` | ✔ | The YDKE URL of the deck you want to display. | text
+`public` | ❌ | Whether to display the deck publicly, or only show it to yourself with an ephemeral reply. | boolean
+`stacked` | ❌ | Whether to display deck sections stacked on top of each other in one long column, or in seperate columns side-by-side. | boolean
```

Comment:
Use `"true" or "false"` since only mathematicians and programmers would understand what a "boolean" is.

Reply:
Or we could give a definition above since we really intend to convey true = yes and false = no